packages/flutter_tools/test/general.shard/web/devfs_web_test.dart

```diff
@@ -1134,6 +1148,8 @@ void main() {
     final WebDevFS webDevFS = WebDevFS(
       hostname: 'localhost',
       port: 0,
+      tlsCertPath: null,
+      tlsCertKeyPath: null,
```

Comment:
Any chance of adding a test where these are not `null`, and we can verify that the server is bound to `https`?

Reply:
This might end up being a little complicated than its purpose since the certificate need to be generated and then passed and then the connection will have to be checked. I can try but it might take sometime since this is not elemetary.

Reply:
@arpitgandhi9 I'd appreciate it if you gave it a fair shot, that way the feature is harder to "revert". If needed, an empty certificate and key can possibly be added as test assets?

Reply:
Just checked this, was caught up in some personal commitments. Will check this asap.

Reply:
@ditman Added test case to validate connection URI with a dummy certificate. This certificate is part of a dummy test asset package.